### PR TITLE
Add disk fields for cs show

### PR DESCRIPTION
**Pull request by kathryn-zhou**
## Changes proposed in this PR
- Add 'Disk Request', 'Disk Limit', and 'Disk Type' fields to cs show

## Why are we making these changes?
- Allow users to see disk specs on jobs they submitted

### CODE REVIEW

File: cli/cook/subcommands/show.py [hunk 1]
```diff
@@ -46,6 +46,12 @@ def tabulate_job(cluster_name, job):
                       ['Priority', job['priority']]]
     if job['max_runtime'] != DEFAULT_MAX_RUNTIME:
         job_definition.append(['Max Runtime', millis_to_timedelta(job['max_runtime'])])
+    if 'disk' in job:
+        job_definition.append(['Disk Request', job['disk']['request']])
+        if 'limit' in job['disk']:
+            job_definition.append(['Disk Limit', job['disk']['limit']])
+        if 'type' in job['disk']:
+            job_definition.append(['Disk Type', job['disk']['type']])
```
Comment (dposada) [inline cli/cook/subcommands/show.py:54]: Could you share an example of what this looks like through `cs show`?

File: integration/tests/cook/test_cli.py [hunk 2]
```diff
@@ -1255,6 +1255,49 @@ def test_show_progress_message_custom_progress_file(self):
...
+        cp = cli.show(uuids, self.cook_url)
+        self.assertEqual(0, cp.returncode, cp.stderr)
+        self.assertIn("Disk Request", cli.stdout(cp))
+        self.assertIn("Disk Limit", cli.stdout(cp))
+        self.assertIn("Disk Type", cli.stdout(cp))
+
+
+
+
+
```
Comment (dposada) [inline integration/tests/cook/test_cli.py:1300]: Too many blank lines

Comment (kathryn-zhou) [reply integration/tests/cook/test_cli.py:1300]: deleted

File: cli/cook/subcommands/show.py (hunk 1 above)
Comment (kathryn-zhou) [reply cli/cook/subcommands/show.py:54]: Screen Shot 2020-11-11 at 11 51 12 AM
Screen Shot 2020-11-11 at 11 52 27 AM
Screen Shot 2020-11-11 at 11 53 18 AM

Comment (dposada) [reply cli/cook/subcommands/show.py:54]: Discussed offline moving GPUs and Disk rows to right under Memory and CPUs

Comment (kathryn-zhou) [reply cli/cook/subcommands/show.py:54]: moved GPUs before Disk rows but leaving the required fields above the optional fields

**Review by nsinkov — APPROVED** [review]

**Review by dposada — APPROVED** [review]

**Pull request merged**